### PR TITLE
Making the method fterm_lazygit a generic function that can be used to call other app as well

### PR DESCRIPTION
**Pull request by colutti**
It can be used to run ranger for example. Just pass the name of the command to run to the function.

### CODE REVIEW

File: lua/lv-floatterm/init.lua [hunk 1]
```diff
@@ -16,41 +16,24 @@ M.config = function()
...
+        height = 0.9,
+        width = 0.9,
+        x = 0.5,
+        y = 0.3,
+      },
+    }
+    lazy:toggle() 
   end
 
-  -- Map esc to exit inside lazygit
```
Comment (abzcoding) [inline lua/lv-floatterm/init.lua:44]: can we keep this for `lazygit` somehow?

File: lua/lv-floatterm/init.lua [hunk 2]
```diff
@@ -16,41 +16,24 @@ M.config = function()
...
-  vim.api.nvim_exec([[
-  function LazyGitNativation()
-    echom &filetype
-    if &filetype ==# 'FTerm'
-      tnoremap <Esc> q
-      tnoremap <C-v><Esc> <Esc>
-    endif
-  endfunction
-  ]], false)
 end
```
Comment (abzcoding) [inline lua/lv-floatterm/init.lua:37]: can we keep this for `lazygit` somehow?

Comment (colutti) [reply lua/lv-floatterm/init.lua:48]: done

**Review by colutti — COMMENTED** [review]
It should be good now.

**Comment by colutti** [comment]
Next step would be to add keymaps for ranger :)

File: lua/lv-which-key/init.lua [hunk 3]
```diff
@@ -23,8 +23,8 @@ which_key.setup {
     },
   },
   icons = {
-    breadcrumb = "»", -- symbol used in the command line area that shows your active key combo
-    separator = "➜", -- symbol used between a key and it's label
+    breadcrumb = "¬ª", -- symbol used in the command line area that shows your active key combo
+    separator = "‚ûú", -- symbol used between a key and it's label
```
Comment (abzcoding) [inline lua/lv-which-key/init.lua:27]: can you revert these two lines, seems like unrelated to this PR

Comment (colutti) [reply lua/lv-which-key/init.lua]: Nice catch. Done.

**Comment by abzcoding** [comment]
seems good to me, thank you @colutti

**Comment by abzcoding** [comment]
R.I.P FTerm 💀
